ibc-core/ics02-client/types/src/status.rs

```diff
@@ -38,6 +40,14 @@ impl Status {
     pub fn is_expired(&self) -> bool {
         *self == Status::Expired
     }
+
+    /// Checks whether the status is active; returns `Err` if not.
+    pub fn verify_active(&self) -> Result<(), ClientError> {
```

Comment:
Let's name this `verify_is_active` to get consistent with the naming of other similar methods (e.g.`Version::verify_is_supported`).

Reply:
Well, my excuse is that ChannelEnd::verify_not_closed doesn’t follow that convention. ;)